src/Kanbas/Courses/Quizzes/Editor/index.tsx

```diff
@@ -1,23 +1,54 @@
+import { FaCheckCircle, FaEllipsisV } from "react-icons/fa";
 import MiddleContent from "../../../layout/Content/NotLeftSide/MiddleContent";
 import MiddleContentActions from "../../../layout/Content/NotLeftSide/MiddleContent/MiddleContentActions";
 import MiddleContentData from "../../../layout/Content/NotLeftSide/MiddleContent/MiddleContentData";
 import QuizDetailsEditor from "./Details";
 import QuizQuestionsEditor from "./Questions";
+import { Button, Tab, Tabs } from "react-bootstrap";
+import { useState } from "react";
+import "./index.css";
 
 function QuizEditor() {
+
+    const [key, setKey] = useState<string>("Details")
```

Comment:
Instead of raw strings, set this up as an enum so it is restricted. This will make converting tabs to a common component as well.